docs/crio.8.md

```diff
@@ -232,6 +233,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
        their CA to their system's list of trusted CAs instead of using
        '--insecure-registry'. (default: [])
 
+**--internal-wipe**: Whether CRI-O should wipe containers on server startup. If set to false, 'crio wipe' will cleanup the containers instead.
```

Comment:
what does "cleanup the containers" mean?  I don't see another instance of "cleanup" in this entire man page.  A bit more detail please.  It would also be good to be specific about what wipe does.

Reply:
alright I've updated the verbiage to hopefully be more clear